packages/stryker/src/reporters/ClearTextScoreTable.ts

```diff
@@ -0,0 +1,167 @@
+import { ScoreResult } from 'stryker-api/report';
+import * as os from 'os';
+import * as _ from 'lodash';
+import * as chalk from 'chalk';
+
+const FILES_ROOT_NAME = 'All files';
+
+interface TableCellValueFactory {
+  (row: ScoreResult, ancestorCount: number): string;
+}
+
+const spaces = (n: number): string => {
+  if (n) {
+    return ` ${spaces(n - 1)}`;
+  } else {
+    return '';
+  }
+};
+
+const dots = (n: number): string => {
+  if (n) {
+    return `.${dots(n - 1)}`;
+  } else {
+    return '';
+  }
+};
+
+/**
+ * Represents a column in the clear text table
+ */
+class Column {
+  protected width: number;
+
+  constructor(public header: string, public valueFactory: TableCellValueFactory, public rows: ScoreResult) {
+    const maxContentSize = this.determineValueSize();
+    this.width = this.pad(dots(maxContentSize), maxContentSize).length;
+  }
+
+  protected determineValueSize(row: ScoreResult = this.rows, ancestorCount: number = 0): number {
+    const valueWidths = row.childResults.map(child => this.determineValueSize(child, ancestorCount + 1));
+    valueWidths.push(this.header.length);
+    valueWidths.push(this.valueFactory(row, ancestorCount).length);
+    return Math.max(...valueWidths);
+  }
+
+  /**
+   * Adds padding (spaces) to the front and end of a value
+   * @param input The string input
+   * @param targetLength The min length that has to be reached
+   */
+  protected pad(input: string, targetLength: number = this.width): string {
+    // End with space
+    if (!input.endsWith(' ')) {
+      return this.pad(input + ' ', targetLength);
+    }
+    // Start with space
+    else if (!input.startsWith(' ')) {
+      return this.pad(' ' + input, targetLength);
+    }
+    else if (input.length < targetLength) {
+      // Pad rest with spaces in front
+      return spaces(targetLength - input.length) + input;
```

Comment:
I don't think so. That won't guarantee that it starts with a space. The code right now really relies on the `pad` function at least adding a space even if it would exceed the target length. ... I hate drawing tables in clear text ;)